test/fuzz/CMakeLists.txt

```diff
@@ -44,6 +44,17 @@ if(NOT libclangfuzzer)
     return()
 endif()
 
+# We require either a PIC-enabled or shared library build to build
+# our dynamic fuzz_util library. Avoid building the fuzz test if not available.
+set(wiredtiger_target)
+if(ENABLE_STATIC AND WITH_PIC)
+    set(wiredtiger_target wiredtiger_static)
+elseif(ENABLE_SHARED)
+    set(wiredtiger_target wiredtiger_shared)
+else()
+    return()
```

Comment:
Will logging an informational message be helpful here?

Reply:
Yep, that's a great point. I've update it with a `STATUS` level message.